scripts/globals/crafting.lua

```diff
@@ -297,15 +297,16 @@ function unionRepresentativeTrade(player, npc, trade, csid, guildID)
         if remainingPoints == 0 then
             player:messageText(npc, NO_MORE_GP_ELIGIBLE);
         else
+            local totalItems = 0;
```

Comment:
Can you format for the new standards? Just the function you are working in is fine, rather than the whole file.

Remove semicolons, space out function variables etc.